packages/cli-doctor/src/tools/versionRanges.ts

```diff
@@ -3,6 +3,7 @@ export default {
   NODE_JS: '>= 14',
   YARN: '>= 1.10.x',
   NPM: '>= 4.x',
+  RUBY: '>= 2.6.x',
```

Comment:
This doesn't seem to agree with the template's .ruby-version file? (At least for the latest release) https://github.com/facebook/react-native/blob/v0.71.2/template/_ruby-version

Reply:
yeah, can we read it from the `.ruby-version` file, instead of hardcoding it? So we don't have to manually maintain two version numbers.

Reply:
> yeah, can we read it from the `.ruby-version` file, instead of hardcoding it? So we don't have to manually maintain two version numbers.

There are a few others that could potentially be sourced from elsewhere too. For example; .node-version, package.json "engines", Gradle files, etc. That should probably be a different PR though.

If this reads from `.ruby-version`, is there any worry about confusion from `doctor` saying 'you need Ruby x' because the `.ruby-version` has been changed and it may not be accurate with compatibility to RN?

Reply:
I think it should not read it from `.ruby-version`, but it should hardcode it on the same version so they're aligned. Users could edit or remove `.ruby-version` so that's a non reliable source.

Reply:
Changed the version range required to be the one in `.ruby-version` file.

Reply:
what if we read the React Native version from the `package.json` and we query GitHub to know which version of ruby we should support? Something like a GET request to this url: `https://github.com/facebook/react-native/blob/${versions}/template/_ruby-version`, replacing the `${version}` placeholder?

Reply:
> and we query GitHub to know which version of ruby we should support?

That's doable but it adds a lot of complexity and doing networking calls inside a CLI should be handled with care (folks can be on proxies, vpns, rate limited, etc.). 

Are you suggesting this because you're afraid we'll forget to update this version? IMHO the hardcoded version here is the more robust solution on the user end. We can have a `prepack` script at publish time of the CLI that runs those checks and flags you if you're about to ship something which is not properly aligned.